homeassistant/components/fibaro/sensor.py

```diff
@@ -17,33 +22,32 @@
 
 from . import FIBARO_DEVICES, FibaroDevice
 
-SENSOR_TYPES = {
-    "com.fibaro.temperatureSensor": [
-        "Temperature",
-        None,
-        None,
-        SensorDeviceClass.TEMPERATURE,
-    ],
-    "com.fibaro.smokeSensor": [
-        "Smoke",
-        CONCENTRATION_PARTS_PER_MILLION,
-        "mdi:fire",
-        None,
-    ],
-    "CO2": [
-        "CO2",
-        CONCENTRATION_PARTS_PER_MILLION,
-        None,
-        None,
-        SensorDeviceClass.CO2,
-    ],
-    "com.fibaro.humiditySensor": [
-        "Humidity",
-        PERCENTAGE,
-        None,
-        SensorDeviceClass.HUMIDITY,
-    ],
-    "com.fibaro.lightSensor": ["Light", LIGHT_LUX, None, SensorDeviceClass.ILLUMINANCE],
+SENSOR_TYPES: dict[str, SensorEntityDescription] = {
+    "com.fibaro.temperatureSensor": SensorEntityDescription(
+        key="Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+    ),
```

Comment:
Adjust the description to something like this:

```suggestion
    SensorEntityDescription(
        key="com.fibaro.temperatureSensor",
        name="Temperature",
        device_class=SensorDeviceClass.TEMPERATURE,
    ),
```